### PR TITLE
Review and Corrections for Condition

### PR DESCRIPTION
**Pull request by RicardoCrespo-ora**
Description
----
In preparation for migrating to Oracle docs, we are reviewing and correcting any inconsistencies, typos, broken links, etc. This PR aims to correct any of the above that needs to be addressed within Consent/Contract and any other information directly referenced by it.

### Broken Links for Condition.category US Core Condition Category Codes

image
*click*
image

**After**
image

---

### Broken Links for Condition.code ICD-10 details 

**Before**
image
*click*
image

**After**
image

---

### Formatting changes to Search parameters and implementation details

**Before**
image

**After**
image
---

### Updated example for Patient Authorization Request For Resolved Status

**Before**
image

**After**
image

---

### Updated example for Patient Authorization Request For Entered in Error Status

**Before**
image

**After**
image

---

### Formatting for Create implementation details

**Before**
image

**After**
image

---

### Formatting for Update implementation details

**Before**
image

**After**
image

---

### Formatting for Create implementation details

**Before**
image

**After**
image

---

### Formatting for Update implementation details

**Before**
image

**After**
image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_condition.rb [hunk 1]
```diff
@@ -102,171 +102,168 @@ module Resources
...
-            'display': 'physician requested information sensitivity',
-            'userSelected': false
-          },
-          {
-            'system': 'http://terminology.hl7.org/CodeSystem/v3-ActCode',
-            'code': 'PRS',
-            'display': 'patient requested information sensitivity',
-            'userSelected': false
-          }
-        ]
```
Comment (AlexandruBasarab) [inline lib/resources/example_json/r4_examples_condition.rb:123]: Why was this removed ? 
Condition has in meta security labels.

**Review by napster235 — DISMISSED** [review]

File: lib/resources/example_json/r4_examples_condition.rb (hunk 1 above)
Comment (napster235) [reply lib/resources/example_json/r4_examples_condition.rb:123]: Can this be put back since we are now supporting the meta.security field?

Comment (RicardoCrespo-ora) [reply lib/resources/example_json/r4_examples_condition.rb]: I have added the security label from the previous example into this one

**Review by AlexandruBasarab — APPROVED** [review]

**Review by napster235 — APPROVED** [review]

**Comment by Someshnsn** [comment]
After Deployment:
Screenshot 2023-11-10 at 7 15 47 PM
Screenshot 2023-11-10 at 7 15 53 PM
Screenshot 2023-11-10 at 7 17 06 PM
Screenshot 2023-11-10 at 7 21 14 PM
Screenshot 2023-11-10 at 7 22 06 PM
Screenshot 2023-11-10 at 7 22 39 PM
Screenshot 2023-11-10 at 7 23 14 PM
Screenshot 2023-11-10 at 7 23 21 PM
Screenshot 2023-11-10 at 7 24 19 PM
Screenshot 2023-11-10 at 7 24 41 PM
